.github/workflows/release.yml

```diff
@@ -188,3 +188,39 @@ jobs:
             ${{ steps.rename_binary.outputs.name }}.tar.gz
             ${{ steps.rename_binary.outputs.name }}.tar.gz.sha256
             assets/Output/*
+
+  publish-winget:
+    needs: release
+    runs-on: windows-latest
+    steps:
+      - if: ${{ github.event_name != 'workflow_run' }}
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: RussellBanks.Komac
+          installers-regex: 'KomacSetup.*\.exe$'
+          token: ${{ secrets.WINGET_TOKEN }}
+      - if: ${{ github.event_name == 'workflow_run' }}
+        name: Get nightly version
+        run: |
+          $cargo = Invoke-WebRequest -Uri https://raw.githubusercontent.com/russellbanks/Komac/main/Cargo.toml
+          $Version = ($cargo | Select-String -Pattern '(?<=version\s=\s")[\d.]+').Matches.Value
+          "version=$Version" >> $env:GITHUB_OUTPUT
+        id: get-version
+      - if: ${{ github.event_name == 'workflow_run' }}
+        uses: vedantmgoyal2009/winget-releaser@v2
+        with:
+          identifier: RussellBanks.Komac.Nightly
+          version: ${{ steps.get-version.outputs.version }}
+          installers-regex: 'KomacSetup.*\.exe$'
+          release-tag: nightly
+          token: ${{ secrets.WINGET_TOKEN }}
+
+  publish-homebrew-tap:
+    needs: release
+    if: ${{ github.event_name != 'workflow_run' }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger update formula workflow (homebrew-tap)
+        run: gh workflow run komac.yml -f version=$($env:GITHUB_REF_NAME.TrimStart('v')) -R russellbanks/homebrew-tap
```

Comment:
I think the shell might need to be powershell (`pwsh`) if you're using `TrimStart`